src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1589,11 +1589,17 @@ def _install_dev_spaces_extension(extension_name):
     return True
 
 
-def _update_dev_spaces_extension(extension_name):
+def _update_dev_spaces_extension(extension_name, extension_module):
     from azure.cli.core.extension import ExtensionNotInstalledException
     try:
         from azure.cli.command_modules.extension import custom
         custom.update_extension(extension_name=extension_name)
+
+        # reloading the imported module to update
+        from importlib import reload as reload
```

Comment:
Hum? Can't it just be `from importlib import reload`?

Reply:
Updated